src/skylink/sia.ts

```diff
@@ -1,10 +1,29 @@
-import { fromByteArray } from "base64-js";
+import base32Decode from "base32-decode";
+
 import { hashAll } from "../crypto";
-import { encodePrefixedBytes } from "../utils/encoding";
+import { base64RawUrlToUint8Array, uint8ArrayToBase64RawUrl, encodePrefixedBytes } from "../utils/encoding";
 import { hexToUint8Array, isASCIIString, stringToUint8ArrayUtf8, trimSuffix } from "../utils/string";
 import { validateHexString, validateNumber, validateUint8ArrayLen } from "../utils/validation";
 
-// The raw size of the data that gets put into a link.
+/**
+ * The size of the Skylink after it has been encoded using base32.
```

Comment:
"string length" might be more clear than "size" here.

Reply:
These comments were copied from skyd. Fixed.